### PR TITLE
Minor changes to welcome and overview sections

### PR DESCRIPTION
**Pull request by pl53**
Minor polishment to make reading smoother.

*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: content/en-us/_index.md [hunk 1]
```diff
@@ -4,14 +4,14 @@ title: "Home"
...
+Many IoT designs now being used globally are well-worn and time-tested, which often come from mature domains such as *sensor networks*, *[control system](https://en.wikipedia.org/wiki/Control_system)* architectures, *[machine-to-machine ](https://en.wikipedia.org/wiki/Machine_to_machine)* networks, and *[embedded system](https://en.wikipedia.org/wiki/Embedded_system)* development. However, the lack of visibility of those mature designs is causing many people to [reinvent the wheel](https://en.wikipedia.org/wiki/Reinventing_the_wheel) for their solution, when they would prefer not. To further complicate the matter for experienced practitioners, even when a long-standing design is known, the design often needs to be revised to take into account cloud development concepts.
```
Comment (brettf) [inline content/en-us/_index.md]: This is just a matter of style. Please revert to original text here.

File: content/en-us/_index.md [hunk 2]
```diff
@@ -4,14 +4,14 @@ title: "Home"
 {{< atlas-logo >}} 
 # Welcome
 
-Welcome to the IoT Atlas, where successful journeys using the internet of things rely on maps.
+Welcome to the IoT Atlas, where successful journeys using the Internet of Things (IoT) rely on maps.
 
-The most successful Internet of Things (IoT) projects have clearly defined business outcomes. The vast majority of IoT projects traverse different phases, starting from prototype, through pilot and limited production toward production, to achieve the identified business outcome at scale. The IoT Atlas supports your project by explaining the why, what, and who of commonly used, modern IoT designs.
+The most successful IoT projects have clearly defined business outcomes. The vast majority of IoT projects traverse different phases, starting from prototype, through pilot and limited production toward production, to achieve the identified business outcome at scale. The IoT Atlas supports your project by explaining the why, what, and who of commonly used, modern IoT designs.
```
Comment (brettf) [inline content/en-us/_index.md]: After a quick chat among the moderators, we agree that changing IoT after the initial mention is more clear.

File: content/en-us/_index.md [hunk 3]
```diff
@@ -4,14 +4,14 @@ title: "Home"
...
+The IoT Atlas is a resource for both new and long-time builders of IoT solutions. It updates and expands upon well-worn designs with the presumption that today, building "Internet of Things" solutions almost always means that "a hyper-scale cloud is available" to practitioners building each solution. Taking the stance that solutions development can rely upon the existence of a hyper-scale cloud does not imply the IoT Atlas' designs must be connected in order to operate. Instead, the designs attempt to leverage the strength of a fleet of devices operating locally in remote environments for long periods of time **and** the cloud's strengths of massive scale, global ubiquity, and solution agility. The designs can then be used to build bar-raising solutions that expect intermittent, long-duration, or primarily-disconnected network outages as a fundamental influence of their designed components. Essentially the designs herein follow the IoT philosophy that *the edge completes the cloud*.
```
Comment (brettf) [inline content/en-us/_index.md]: This propagation of "IoT" in place of "internet of things" aligns. Thanks for the change.

File: content/en-us/_index.md [hunk 4]
```diff
@@ -41,4 +41,4 @@ Finally, each design will follow a few conventions. To convey an example of eith
 ## Team
 At this time, the core maintainers of the IoT Atlas are [Brett Francis](https://github.com/brettf), 
 [Gavin Adams](https://github.com/gadams999), and 
-[Craig Williams](https://github.com/typemismatch). We're excited to get these designs out into the world here and on [GitHub](https://github.com/aws/iot-atlas); so together we can accelerate IoT progress with you.   
\ No newline at end of file
+[Craig Williams](https://github.com/typemismatch). We're excited to get these designs out into the world here and on [GitHub](https://github.com/aws/iot-atlas); so together we can accelerate IoT progress with you.
```
Comment (brettf) [inline content/en-us/_index.md]: Can't quite tell what this change is. It looks like no change (except maybe newline?)

Comment (pl53) [reply content/en-us/_index.md]: I think it is just the new line.

File: content/en-us/_index.md (hunk 1 above)
Comment (pl53) [reply content/en-us/_index.md]: No objection, but just some nitpick, I prefer "would not" over "wouldn't" as this is formal writing.

**Pull request merged**